src/data/data_split.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+
+
+def train_test_split(data_path: str="../data/raw/", test_size: float=.3,
+                     save_files: bool=True):
+    df = pd.read_parquet(data_path + 'final_taxi_data.parquet').sort_values('pickup_datetime').reset_index(drop=True)
```

Comment:
I would definitely put file name (final_data etc) as an argument, so we can run main.py with config file containing code-specific info and keep code clean with the least amount of dependencies. To be discussed as always